docs/auth/social-auth.md

```diff
@@ -312,10 +312,14 @@ async function onGoogleButtonPress() {
   // Check if your device supports Google Play
   await GoogleSignin.hasPlayServices({ showPlayServicesUpdateDialog: true });
   // Get the users ID token
-  const { idToken } = await GoogleSignin.signIn();
+  const signInResult = await GoogleSignin.signIn();
+
+  if (!signInResult.data?.idToken) {
+    throw new Error("No ID token found");
+  }
```

Comment:
This is not backwards compatible, with no documentation indicating what minimum version uses this style
Perhaps it could be:

```suggestion
  // Try the new style of google-sign in result, from v13+ of that module
  idToken = signInResult.data?.idToken;
  if (!idToken) {
      // if you are using older versions of google-signin, try old style result
      idToken = signInResult.idToken;
  }
  if (!idToken) {
    throw new Error("No ID token found");
  }
```